DigitalLearningSolutions.Data/Models/Courses/CourseNameInfo.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace DigitalLearningSolutions.Data.Models.Courses
+{
+    public class CourseNameInfo

```

Comment:
There's a CourseDetails class that has the naming logic. To avoid unit tests for all different classes, we should make it extend this base class (but use the CourseDetails namings for the properties)